.github/renovate.json

```diff
@@ -1,3 +1,30 @@
 {
-  "extends": ["github>SocialGouv/renovate-config:light"]
+  "extends": ["github>SocialGouv/renovate-config:light"],
+  "ignorePresets": ["group:allNonMajor"],
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchDepTypes": ["dependencies"],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "groupName": "all non-major dependencies",
+      "groupSlug": "all-dependencies-minor-patch"
+    },
+    {
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchDepTypes": ["devDependencies"],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "groupName": "all non-major dev dependencies",
+      "groupSlug": "all-devdependencies-minor-patch"
+    }
+  ]
```

Comment:
Already merged in [`github>SocialGouv/renovate-config:light`](https://github.com/SocialGouv/renovate-config/commit/0ea04bbeab54e14117a1016e07d75fc04abc97d6)